packages/beacon-node/test/unit/util/sszBytes.test.ts

```diff
@@ -84,7 +87,7 @@ describe("attestation SSZ serialized picking", () => {
   it("getAttDataBase64FromAttestationSerialized - invalid data", () => {
```

Comment:
`getAttDataFromAttestationSerialized - invalid data` might be better to match the name of the function that's being tested